tests/test_sensor_jira_sensor.py

```diff
@@ -4,36 +4,50 @@
 
 from jira_sensor import JIRASensor
 
-JIRA_URL = "https://ja.atlassian.com/"
-
-MOCK_PAYLOAD = {
-    "issue_name": "ISSUEKEY-1",
-    "issue_url": "https://ja.atlassian.com/api/hoge",
-    "issue_browse_url": JIRA_URL + "/browse/ISSUEKEY-1",
-    "project": "PROJECT",
-    "created": "2018-11-12T11:20:54.000+0900",
-    "assignee": "user01",
-    "fix_versions": ["verA"],
-    "issue_type": "task"
-}
-
-MOCK_TRIGGER = {
-    "trace_context": None,
-    "trigger": "jira.issues_tracker",
-    "payload": MOCK_PAYLOAD
-}
+JIRA_URL = "http://jira.hoge.com/"
```

Comment:
Hmmm, this basically completely rewrites the test further down, which I really don't like. I'm presuming that the existing mocked payload is similar to a realistic/actual payload at some point, and we should support both that payload as well as the payload from the "new" v2 REST API.